### PR TITLE
Abuida - Added page descriptions, login and logout tests

### PR DESCRIPTION
**Pull request by AlexBuida**

### CODE REVIEW

File: LiveCodingUi/Tests/LoginTests.cs [hunk 1]
```diff
@@ -27,12 +27,12 @@ public async Task LoginAsStandardUserWithValidCredentials()
...
-            await loginPage.UsernameInput.FillAsync("standard_user");
-            await loginPage.PasswordInput.FillAsync("secret_sauce");
-            await loginPage.LoginButton.ClickAsync();
+            await loginPage.Login("standard_user", "secret_sauce");
 
             //assert
-            //add verification steps that inventory page is displayed
+            var inventoryPage = new InventoryPage(Page);
+            bool isDisplyed = await inventoryPage.IsInventoryPageDisplayed();
+            Assert.That(isDisplyed);
```
Comment (Lickut) [inline LiveCodingUi/Tests/LoginTests.cs]: Assert.That(isDisplayed, Is.True)

File: LiveCodingUi/Tests/LogoutTests.cs [hunk 2]
```diff
@@ -20,17 +21,21 @@ public async Task Setup()
...
-            //add logout steps by opening side panel and clicking Logout
+            var inventoryPage = new InventoryPage(Page);
+            await inventoryPage.OpenSideMenu();
+            await inventoryPage.ClickLogout();
 
             //assert
-            //add verification steps that loging page is displayed
-            throw new NotImplementedException();
+            bool isDisplayed = await loginPage.IsLoginPageDisplayed();
+            Assert.That(isDisplayed);
```
Comment (Lickut) [inline LiveCodingUi/Tests/LogoutTests.cs]: Assert.That(isDisplayed, Is.True)

File: LiveCodingUi/Pages/InventoryPage.cs [hunk 3]
```diff
@@ -10,5 +10,27 @@ public InventoryPage(IPage page)
...
+
+        public async Task<bool> IsInventoryPageDisplayed() => await SideMenuButton.IsVisibleAsync();
+        public async Task OpenSideMenu() => await SideMenuButton.ClickAsync();
+        public async Task OpenAllItems() => await AllItemsLink.ClickAsync();
+        public async Task OpenAbout() => await AboutLink.ClickAsync();
+        public async Task ClickLogout() => await LogoutLink.ClickAsync();
+        public async Task ClickResetAppStateLink() => await ResetAppStateLink.ClickAsync();
+        public async Task CloseSideMenu() => await SideMenuButton.ClickAsync();
+        public async Task OpenCart() => await CartButton.ClickAsync();
+        public async Task SortNameAZ() => await SortingDropdown.SelectOptionAsync("az");
```
Comment (Lickut) [inline LiveCodingUi/Pages/InventoryPage.cs:31]: What will happen with this option if another language will be selected in application?

Comment (AlexBuida) [reply LiveCodingUi/Pages/InventoryPage.cs:31]: This site has only one language.
But in case site has several languages, we can use indexes in dropdown

**Pull request closed without merging**